test/specs/components/Input/Input-test.ts

```diff
@@ -1,10 +0,0 @@
-import * as React from 'react'
-
-import { isConformant } from 'test/specs/commonTests'
-import { getTestingRenderedComponent } from 'test/utils'
-
-import Input from 'src/components/Input/Input'
-
-describe('Input', () => {
-  isConformant(Input)
-})
```

Comment:
**💡 Tip**

You can use `git mv <old-path> <new-path>` to move files.  This will ensure git can track changes between filenames instead of appearing as a file delete and and file create.  It improves readability and also makes rebase/merge more robust.